infra/gcb/build.py

```diff
@@ -199,8 +199,18 @@ def get_build_steps(project_yaml, dockerfile_path):
               # We also remove /work and /src to save disk space after a step.
               # Container Builder doesn't pass --rm to docker run yet.
               'rm -r /out && cd /src && cd {1} && mkdir -p {0} && compile && rm -rf /work && rm -rf /src'.format(out, workdir),
-              ],
-            },
+            ],
+           },
+          # test binaries
+          {'name': 'gcr.io/oss-fuzz-base/base_runner',
+            'env': env,
+            'args': [
+              'bash',
+              '-c',
+              'test_all && ' # To verify that fuzzers generally work.
+              'test_bad_build' # To verify instrumentation and other issues.
```

Comment:
`test_bad_build` is not implemented, the approach described in the comments in each test project